azure-pipelines.yml

```diff
@@ -1,342 +1,396 @@
+                  overrideAction: ${{ parameters.action }}
+                  environment: ${{ component.env }}
+                  component: managed-identity
+                  location: ${{ parameters.location }}
+                  serviceConnection: ${{ component.serviceConnection }}
+                  terraformInitSubscription: ${{ variables.tfInitSub }}
+                  product: ${{ variables.product }}
+                  tfVarsFile: "$(System.DefaultWorkingDirectory)/$(buildRepoSuffix)/environments/managed-identity/${{ component.env }}.tfvars"
+                  initCommandOptions: >
+                    -var subscription_id=${{ component.terraformSubscriptionID }}
+                    -backend-config resource_group_name=${{ component.backendAzureRmResourceGroupName }}
+                  planCommandOptions: >
+                    -var subscription_id=${{ component.terraformSubscriptionID }}
+                    -var control_vault=$(controlKeyVault)
+                    -var project=$(project)
 
-  - stage: Managed_Identity
-    displayName: 'Managed_Identity'
-    dependsOn: Network
-    jobs:
-      - job: Managed_Identity
-        steps:
-          - template: steps/terraform.yaml@cnp-azuredevops-libraries
-            parameters:
-              overrideAction: ${{ parameters.action }}
-              environment: ${{ parameters.env }}
-              component: managed-identity
-              location: ${{ parameters.location }}  
-              serviceConnection: $(serviceConnection)
-              terraformInitSubscription: ${{ variables.tfInitSub }}
-              product: ${{ variables.product }}
-              tfVarsFile: "$(System.DefaultWorkingDirectory)/$(buildRepoSuffix)/environments/managed-identity/${{ parameters.env }}.tfvars"
-              initCommandOptions: >
-                -var subscription_id=${{ variables.terraformSubscriptionID }}
-                -backend-config resource_group_name=$(backendAzureRmResourceGroupName)
-              planCommandOptions: >
-                -var subscription_id=${{ variables.terraformSubscriptionID }}
-                -var control_vault=$(controlKeyVault)
-                -var project=$(project)
+      - stage: "Aks_${{ component.env }}"
+        displayName: "${{ upper(component.env) }}: AKS clusters"
+        dependsOn:
+          - "CheckingClusters_${{ component.env }}"
+          - "Managed_Identity_${{ component.env }}"
+        variables:
+          existing_clusters: $[stageDependencies.CheckingClusters_${{ component.env }}.CheckingClusters.outputs['check_clusters.cluster_deploy']]
+          deploy_cluster: ${{ parameters.cluster }}
+        condition: |
+            or(
+              and(
+                eq(variables['Build.Reason'], 'Manual'),
+                in('${{ parameters.cluster }}', 'All', '00', '01'),
+                in(dependencies.CheckingClusters_${{ component.env }}.outputs['CheckingClusters.check_clusters.cluster_deploy'], 'All', '00', '01')
```

Comment:
```suggestion

```

Do you need to check this, don't think it's possible to have that value as anything else is it?

Reply:
we have to specified a value after    in(dependencies.CheckingClusters_${{ component.env }}.outputs['CheckingClusters.check_clusters.cluster_deploy'], since we are running it against all value i put all of them there

Reply:
Is it checking anything explicitly though / trying to stop something from happening?